Assets/Scripts/Player/PlayerInteract.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerInteract : MonoBehaviour
+{
+	[SerializeField] private GameObject playerHand;
+
+	private PlayerHandDetection playerHandDetection;
+	private GameObject interactItem;
+	private GameObject heldBox;
+
+	private bool isCarryingItem = false;
+
+
+	private void Start() {
+		playerHandDetection = playerHand.GetComponent<PlayerHandDetection>();
+	}
+
+	// Lever does not change animation-clip
+	private void Update() {
+		if (Input.GetButtonDown("Interact")) {
+			if (playerHandDetection.detectedInteractItem != null && !isCarryingItem) {
+				if (playerHandDetection.detectedInteractItem.name != "PressureButton") {
+					interactItem = playerHandDetection.detectedInteractItem;
+
+					if (interactItem.tag == "Box") {
+						PickUpItem();
+					}
+					else if (interactItem.tag == "Lever" && interactItem.GetComponent<Lever>().isActive == false) {
+						interactItem.GetComponent<Lever>().ActivateLever();
+					}
+					else if (interactItem.tag == "Lever" && interactItem.GetComponent<Lever>().isActive == true) {
+						interactItem.GetComponent<Lever>().DeactivateLever();
+					}
+				}
+			}
+			else if (isCarryingItem) {
+				DropItem();	
+				print("dropping item");
+
+			}
+			
+			print(interactItem);
+		}
+	}
+
+	private void PickUpItem() {
+		isCarryingItem = true;
+		interactItem.GetComponent<Box>().boxCollider.enabled = false;
+		interactItem.GetComponent<Rigidbody2D>().isKinematic = true;
+		interactItem.transform.parent = playerHand.transform;
+		interactItem.transform.localPosition = Vector3.zero;
+	}
+
+	private void DropItem() {
+		isCarryingItem = false;
+		interactItem.GetComponent<Box>().boxCollider.enabled = true;
+		interactItem.GetComponent<Rigidbody2D>().isKinematic = false;
+		heldBox = interactItem;
+		heldBox.transform.parent = null;
+		interactItem = null;
+		heldBox = null;
```

Comment:
hm, this is hard to explain, but I was using interactItem first, but it can become another GameObject in a situation like when the player is currently holding an item and the Button is present in the playerHand-trigger, making it impossible to place the item(box) down. This made it work as intended, but there might be a better way..